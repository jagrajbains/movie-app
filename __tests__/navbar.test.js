import React from "react";
import {shallow} from "enzyme";
import renderer from "react-test-renderer";
import Navbar from "../components/navbar";

describe("Testing the Navbar component", () => {
    it("It should render the Navbar without any errors", () => {
        const component = renderer.create(<Navbar />);
        const tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});