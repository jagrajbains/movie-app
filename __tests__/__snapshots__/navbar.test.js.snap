// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing the Navbar component It should render the Navbar without any errors 1`] = `
<nav
  className="navbar navbar-expand-lg navbar-dark bg-dark fixed-top"
>
  <div
    className="container"
  >
    <a
      className="navbar-brand"
      href="#"
    >
      Start Bootstrap
    </a>
    <button
      aria-controls="navbarResponsive"
      aria-expanded="false"
      aria-label="Toggle navigation"
      className="navbar-toggler"
      data-target="#navbarResponsive"
      data-toggle="collapse"
      type="button"
    >
      <span
        className="navbar-toggler-icon"
      />
    </button>
    <div
      className="collapse navbar-collapse"
      id="navbarResponsive"
    >
      <ul
        className="navbar-nav ml-auto"
      >
        <li
          className="nav-item active"
        >
          <a
            className="nav-link"
            href="#"
          >
            Home
            <span
              className="sr-only"
            >
              (current)
            </span>
          </a>
        </li>
        <li
          className="nav-item"
        >
          <a
            className="nav-link"
            href="#"
          >
            About
          </a>
        </li>
        <li
          className="nav-item"
        >
          <a
            className="nav-link"
            href="#"
          >
            Services
          </a>
        </li>
        <li
          className="nav-item"
        >
          <a
            className="nav-link"
            href="#"
          >
            Contact
          </a>
        </li>
      </ul>
    </div>
  </div>
</nav>
`;
